modules/representation_management/app/serializers/representation_management/original_entities/representative_serializer.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module RepresentationManagement
+  module OriginalEntities
+    class RepresentativeSerializer
+      include JSONAPI::Serializer
+
+      attributes :first_name, :last_name, :full_name,
+                 :address_line1,
+                 :address_line2, :address_line3, :address_type,
+                 :city, :country_name, :country_code_iso3, :province,
+                 :international_postal_code, :state_code, :zip_code, :zip_suffix, :email
+
+      attribute :phone, &:phone_number
+
+      attribute :individual_type do |object|
+        object.user_types.first
```

Comment:
The front end is expecting a single value, not an array because of how `accredited_individuals` model is set up. There really shouldn't be multiple types for one representative as each type would have their own accreditation (a new poa code).